client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx

```diff
@@ -68,24 +75,36 @@ const getExpiryColumn = (
   },
 ];
 
-const NumberOfPacksCell: React.FC<CellProps<DraftInboundLine>> = ({
+const NumberOfPacksCell = ({
   rowData,
   ...props
-}) => (
+}: CellProps<DraftInboundLine>) => (
   <NumberInputCell
     {...props}
     isRequired={rowData.numberOfPacks === 0}
     rowData={rowData}
   />
 );
 
-export const QuantityTableComponent: FC<TableProps> = ({
+export const QuantityTableComponent = ({
   lines,
   updateDraftLine,
   isDisabled = false,
-}) => {
+  item,
+  hasLinkedShipment,
+}: TableProps) => {
+  const t = useTranslation();
   const theme = useTheme();
+  const { data: preferences } = usePreference(
+    PreferenceKey.DisplayVaccineInDoses
+  );
```

Comment:
Will need to refactor once store pref PR is merged >.> using it as a dummy global pref for now